flixel/math/FlxVector.hx

```diff
@@ -96,19 +96,18 @@ class FlxVector extends FlxPoint
 		return this;
 	}
 	
-	/**
-	 * Scale this vector.
-	 * 
-	 * @param	k - scale coefficient
-	 * @return	scaled vector
-	 */
-	public inline function scale(k:Float):FlxVector
-	{
-		x *= k;
-		y *= k;
-		return this;
-	}
-	
+    /**
+     * Scale this vector.
+     * 
+     * @param   k - scale coefficient
+     * @return  scaled vector
+     */
+    public inline override function scale(k:Float):FlxVector
```

Comment:
Oh no. both with override or without override failed.
I tried this:

```
class A{
    public static function main(){
        new B().a();
    }
}

class B extends C {
    public **override** inline function a():B {
        return this;
    }
}

class C {
    public inline function a():C {
        return this;
    }
}
```
with `override`: Field a is inlined and cannot be overridden
without: Field a should be declared with 'override' since it is inherited from superclass C